pkg/appconsts/app_consts.go

```diff
@@ -19,12 +19,18 @@ const (
 	// SubtreeRootThreshold.
 	//
 	// The rationale for this value is described in more detail in ADR-013.
-	SubtreeRootThreshold int    = 64
-	TxSizeCostPerByte    uint64 = 10
-	GasPerBlobByte       uint32 = 8
-	MaxTxSize            int    = 8_388_608 // 8 MiB in bytes
-	TimeoutPropose              = time.Millisecond * 3500
-	TimeoutCommit               = time.Millisecond * 4200
+	SubtreeRootThreshold  int    = 64
+	TxSizeCostPerByte     uint64 = 10
+	GasPerBlobByte        uint32 = 8
+	MaxTxSize             int    = 8_388_608 // 8 MiB in bytes
+	TimeoutPropose               = time.Millisecond * 5000
+	TimeoutProposeDelta          = time.Millisecond * 2000
+	TimeoutPrevote               = time.Millisecond * 1000
+	TimeoutPrevoteDelta          = time.Millisecond * 5000
+	TimeoutPrecommit             = time.Millisecond * 1000
+	TimeoutPrecommitDelta        = time.Millisecond * 5000
+	TimeoutCommit                = time.Millisecond * 200
+	TimeoutHeightDelay           = time.Millisecond * 6000
```

Comment:
What is `TimeoutHeightDelay` used for here? I don't see it called anywhere. What happened to DelayedPrecommitTimeout?